src/common/Models/Kubernetes/Pod.cs

```diff
@@ -1,95 +0,0 @@
-﻿// --------------------------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-// --------------------------------------------------------------------------------------------
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Microsoft.BridgeToKubernetes.Common.Json;
-using Microsoft.BridgeToKubernetes.Common.Logging;
-
-namespace Microsoft.BridgeToKubernetes.Common.Models.Kubernetes
-{
-    /// <summary>
-    /// Kubernetes Pod
-    /// </summary>
-    public class Pod

```

Comment:
@Eneuman This file was deleted intentionally ? I see it doesn't have any usage but maybe we can do another PR to use remove such files ? a PR to remove unused files. Just a thought, please let me know thank you.